cypress/e2e/spec.cy.ts

```diff
@@ -0,0 +1,97 @@
+describe('Flow Tests', () => {
+  it('Shows the landing page', () => {
+    cy.visit('/')
+    cy.get('[data-testid=get_started_button]').click()
+  })
+
+  it('Navigates through the Medicaid only flow', () => {
+    const incomeName = "Suzy"
+    const incomeDescription = "Yardwork"
+    const incomeAmount = '55'
+    const expenseName = 'Gas'
+    const expenseDate = '09/04/2024'
+    const expenseAmount = '33'
+    const username = 'Jane Doe'
+    cy.visit('/')
+
+    // Landing Page
+    cy.get('button').contains('Get Started').should('exist')
+    cy.get('[data-testid=get_started_button]').should('exist')
+    cy.get('[data-testid=get_started_button]').click()
+
+    // How This Works
+    cy.url().should('include', '/introduction/how-this-works')
+    cy.contains("Lorem ipsum").not('be.visible')
```

Comment:
it'll be interesting to see how we evolve on `.contains('all teh text')` v `data-testid="whatever"`. in the past i've indexed on words on screen (cuz that gives us the added bonus of detecting unintentional word changes. however, there's a maintenance burden in changing things in two places. 

also, curious in the `.contains()` side how we account for translation. possible the tradeoff here is different than it was in my other projects. 

no changes requested here, more thinking out loud